index.html

```diff
@@ -1,96 +1,126 @@
-
-The Villas bring to the table win-win survival strategies to ensure proactive domination. At the end of the day, going forward, a new normal that has evolved from generation X is on the runway heading towards a streamlined cloud solution. 
-
-Capitalize on low hanging fruit to identify a ballpark value added activity to beta test. Override the digital divide with additional clickthroughs from DevOps. Nanotechnology immersion along the information highway will close the loop on focusing solely on the bottom line.
-
-OUTSKIRTS
-
-The Outskirts are amazing to the table win-win survival strategies to ensure proactive domination. At the end of the day, going forward, a new normal that has evolved from generation X is on the runway heading towards a streamlined cloud solution. 
-
-Capitalize on low hanging fruit to identify a ballpark value added activity to beta test. Override the digital divide with additional clickthroughs from DevOps. Nanotechnology immersion along the information highway will close the loop on focusing solely on the bottom line.
-
-THE BLOCKS
-
-The Blocks are amazing to the table win-win survival strategies to ensure proactive domination. At the end of the day, going forward, a new normal that has evolved from generation X is on the runway heading towards a streamlined cloud solution. 
-
-Capitalize on low hanging fruit to identify a ballpark value added activity to beta test. Override the digital divide with additional clickthroughs from DevOps. Nanotechnology immersion along the information highway will close the loop on focusing solely on the bottom line.
-
--->
-
-<!-- FOOTER CONTENT
-Interested in starting a project?  
-Let’s talk:
-
-Enter email
-
-We'll never share your email with anyone else.
-
-New York
-123 Lane
-Suite 100
-Albany, NY 12345
-202 555 0144
-
-Florida
-Ocean Drive
-Suite 201
-Orlando, FL 22345
-502 555 0144
-
-California
-Mountain Street
-Suite 105
-San Diego, CA 22345
-702 555 0144
-
-Copyright © 2018 Smith and Jones
--->
-	<script src="js/index.js"></script>
+	<header>
+		<div class="header-content">
+			<h1>S&J</h1>
+			<div class="menu">
+				<img src="./img/nav-hamburger.png" alt="Hamburger Menu Icon" class='menu-button'>
+				<div class="menu-content menu-hidden">
+					<div class="menu-header">
+						<h1>S&J</h1>
+						<img src="./img/nav-hamburger-close.png" alt="Closing X Icon" class="menu-close">
+					</div>
+					<nav>
+						<a class="menu-link" href="./index.html">Home</a>
+						<a class="menu-link" href="./services.html">Services</a>
+						<a class="menu-link" href="./contact.html">Contact</a>
+					</nav>
+				</div>
+			</div>
+		</div>
+	</header>
+	<div class="container home">
+		<div class="cta">
+			<img class="home-jumbo" src="./img/home/home-jumbotron.png" alt="">
+			<img class="home-jumbo-mobile" src="./img/home/home-mobile-jumbotron.png" alt="">
+			<h2>Integrity,<br>Excellence,<br>Progress.</h2>
+		</div>
+		<div class="main-content">
+			<section class="content-top home">
```

Comment:
You probably don't need to apply home here as well, you have it on the outer container.